### PR TITLE
Add record count in dns record table

### LINKED ISSUE #111: Indicate how many more DNS records a user can create in the UI
In #110 we're adding a limit to the number of DNS records a user can have.  We should indicate how many more records a user can create before hitting this limit in the UI.  This will help users understand what is happening with their account.

### PR DESCRIPTION
**Pull request by Genne23v**
Close #111 

Add user DNS record count / limit in DNS table. 

Screenshot 2023-04-03 at 2 13 30 PM

### CODE REVIEW

**Comment by humphd** [comment]
I would flip the order: put the text on the left, button on the right.

You could also play with https://chakra-ui.com/docs/components/stat/usage, which might look nice.

**Comment by Genne23v** [comment]
I updated a bit more styling. Here's what it looks. 

Screenshot 2023-04-04 at 11 27 39 AM


Screenshot 2023-04-04 at 11 28 22 AM

**Comment by Genne23v** [comment]
Here's the new look! 

Screenshot 2023-04-04 at 11 35 17 AM

File: app/routes/__index/dns-records/index.tsx [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+  StatLabel,
+  StatNumber,
+  Text,
+} from '@chakra-ui/react';
 import { Link } from '@remix-run/react';
 import { typedjson, useTypedLoaderData } from 'remix-typedjson';
 import { json } from '@remix-run/node';
 import { z } from 'zod';
 import { parseFormSafe } from 'zodix';
+// import dotenv from 'dotenv';
```
Comment (humphd) [inline app/routes/__index/dns-records/index.tsx:18]: Please remove this.

File: app/routes/__index/dns-records/index.tsx [hunk 2]
```diff
@@ -22,7 +34,13 @@ export type DnsRecordActionIntent = 'renew-dns-record' | 'delete-dns-record';
 export const loader = async ({ request }: LoaderArgs) => {
   const username = await requireUsername(request);
 
-  return typedjson(await getDnsRecordsByUsername(username));
+  return typedjson({
+    dnsRecords: await getDnsRecordsByUsername(username),
+    userDnsRecordCount: await getDnsRecordCountByUsername(username),
+    userDnsRecordLimit: process.env.USER_DNS_RECORD_LIMIT
```
Comment (humphd) [inline app/routes/__index/dns-records/index.tsx:40]: `userDnsRecordLimit: process.env.USER_DNS_RECORD_LIMIT ?? 'unlimited',`

**Review by humphd — CHANGES_REQUESTED** [review]

**Comment by Eakam1007** [comment]
This is just a personal opinion, but would it be better to change the text to something like `You have 1/20 maximum DNS Records`

**Comment by Genne23v** [comment]
> This is just a personal opinion, but would it be better to change the text to something like `You have 1/20 maximum DNS Records`

That would look like this. 

Screenshot 2023-04-04 at 1 07 51 PM

@humphd @Eakam1007  let me know which one is preferred.

**Comment by Eakam1007** [comment]
Hmm, the other style was definitely easier to read

**Comment by Genne23v** [comment]
> Hmm, the other style was definitely easier to read

Thanks for the suggestion and quick feedback. I will keep `Stat` layout as I feel the same.

**Review by dadolhay — APPROVED** [review]

**Pull request merged**